fig_test.go

```diff
@@ -259,6 +268,7 @@ func Test_fig_Load_Defaults(t *testing.T) {
 				want.Logger.Production = false
 				want.Logger.Metadata.Keys = []string{"ts"}
 				want.Application.BuildDate = time.Date(2020, 1, 1, 12, 0, 0, 0, time.UTC)
+				want.Listener = ListenerTCP
```

Comment:
We could not add `Listener` to server.yaml/json/toml and this would be testing the actual default logic. This would also mean we don't have to modify the `_Strict` test.